Plugins/DSLCompletion/DSLComplete.cs

```diff
@@ -255,6 +255,8 @@ private string FindSelection(Check isAllowedChar, bool checkRight = true)
             
             var text = sci.GetLine(sci.CurrentLine);
 
+            if (!text.Contains("\"")) return null;
```

Comment:
Yeah, I just saw it myself. I did it yesterday and forgot about it.
It has nothing to do with the console plugin.